setup.cfg

```diff
@@ -17,10 +17,6 @@ classifiers =
     Operating System :: POSIX
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
```

Comment:
This was formatted by [`setup-cfg-fmt`](https://results.pre-commit.ci/run/github/37489525/1659395979.0dgQqJoTREScUnLfFx-y3g)... not sure if this is expected or a bug?

Reply:
yeah this was an intentional change in 2.0.0 -- if you want these back there's an option `--include-version-classifiers` -- but I'm removing them from all of my projects since they're (1) noisy (2) invite spammy PRs (3) are redundant to `python_requires` and (4) make it more manual work to drop old python versions -- I have no opinions on what pytest chooses though

Reply:
They're probably useless and redundant, but at this point who knows what's using them, so I say keep them.

Reply:
https://pyreadiness.org/3.11/ is using them

Reply:
Added `--include-version-specifiers` to `setup-cfg-fmt` then.